script/Ciel/Day 13/13_01_01_CIEL13_5.txt

```diff
@@ -489,7 +490,7 @@ But I can't put up with any more of this vampire's ranting.
 [sha:72c321d9214e58962e8996d52051045466745837]{
 -- Page 546, Offset 34839.
 -- アイツは吸血鬼です、周りをよく見てください！」
-She's a vampire! Take a good look around you!"
+She's a vampire! I mean just look around you!"
```

Comment:
'I mean' is redundant imo

Reply:
I disagree but think there should be a comma after "mean"